cmd/podman/kube/down.go

```diff
@@ -0,0 +1,38 @@
+package pods
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/spf13/cobra"
+)
+
+var (
+	downDescription = `Command reads in a structured file of Kubernetes YAML.
+   
+  It removes pods that have been based on the Kubernetes kind described in the YAML.`
+
+	downCmd = &cobra.Command{
+		Use:               "down KUBEFILE|-",
+		Short:             "Remove pods based on Kubernetes YAML.",
+		Long:              downDescription,
+		RunE:              down,
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: common.AutocompleteDefaultOneArg,
+		Example:           `podman kube down nginx.yml`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: downCmd,
+		Parent:  kubeCmd,
+	})
+}
+
+func down(cmd *cobra.Command, args []string) error {
+	yamlfile := args[0]
+	if yamlfile == "-" {
+		yamlfile = "/dev/stdin"
```

Comment:
Can you add a test for this bit?

Reply:
This will not work on windows, we should use os.Stdin and pass it along as reader not filepath.